lua/neotest-vitest/init.lua

```diff
@@ -9,8 +9,9 @@ local util = require("neotest-vitest.util")
 ---@field vitestConfigFile? string|fun(): string
 ---@field env? table<string, string>|fun(): table<string, string>
 ---@field cwd? string|fun(): string
+---@field filter_dir? fun(name: string): boolean
```

Comment:
maybe we should add relpath and root parameters

Reply:
Yeap, my bad, done!